### PR TITLE
Only add Sybase test if sybase dir/mount exists

### PR DESCRIPTION
**Pull request by jeanconn**
Only add Sybase test if sybase dir/mount exists.

### CODE REVIEW

**Comment by jeanconn** [comment]
Not sure if this should be a more explicit pytest skip.

File: Chandra/cmd_states/tests/test_get_cmd_states.py [hunk 1]
```diff
@@ -45,7 +46,7 @@ def test_get_states_main():
 
 # Set up possible backends
 dbis = ['hdf5', 'sqlite']
-if six.PY2:
+if six.PY2 and os.path.exists(os.environ['SYBASE']):
```
Comment (taldcroft) [inline Chandra/cmd_states/tests/test_get_cmd_states.py]: Might be a good idea to check `'SYBASE' in os.environ` before this.

**Comment by taldcroft** [comment]
An explicit skip would be a good thing but it is a little annoying to do in this case and not really worth the effort.  At some point we can just drop Sybase support for this.

File: Chandra/cmd_states/tests/test_get_cmd_states.py (hunk 1 above)
Comment (jeanconn) [inline Chandra/cmd_states/tests/test_get_cmd_states.py]: I thought that was a guarantee in any SKA environment, but that would be a good extra check in this line.

Comment (taldcroft) [inline Chandra/cmd_states/tests/test_get_cmd_states.py]: We might one day move away from defining an env var for something that doesn't exist.

**Pull request merged**